ingestion/src/metadata/profiler/profiler/interface/profiler_protocol.py

```diff
@@ -128,17 +128,13 @@ def get_profile_sample_config(
                 profile_sample=entity_config.profileSample,
                 profile_sample_type=entity_config.profileSampleType,
             )
-
-        try:
-            profile_sample = entity.tableProfilerConfig.profileSample
-        except AttributeError:
-            pass
-        else:
-            if profile_sample:
-                return ProfileSampleConfig(
-                    profile_sample=entity.tableProfilerConfig.profileSample,
-                    profile_sample_type=entity.tableProfilerConfig.profileSampleType,
-                )
+        if hasattr(entity.tableProfilerConfig, "profileSample") and hasattr(
+            entity.tableProfilerConfig, "profileSampleType"
+        ):
+            return ProfileSampleConfig(
+                profile_sample=entity.tableProfilerConfig.profileSample,
+                profile_sample_type=entity.tableProfilerConfig.profileSampleType,
+            )
```

Comment:
can we keep the `try/except/else` block, I find it more readable. It also has logic to check if `entity.tableProfilerConfig.profileSample` is None

Reply:
resolved